src/main.rs

```diff
@@ -22,7 +22,7 @@ fn bind_mount(source: &Path, dest: &Path) {
         Some(source),
         dest,
         Some("none"),
-        MsFlags::MS_BIND | MsFlags::MS_REC,
+        MsFlags::MS_BIND | MsFlags::MS_REC | MsFlags::MS_PRIVATE,
```

Comment:
Mhm. Does this even matter since we clone into a mountnamespace with CLONE_NEWNS?

Reply:
Or where does this become a problem?

Reply:
At least on bionic, if you run `mount` outside of the created namespace, all of the mounts are visible -- which isn't really useful information for administrators of the machine.